.travis.yml

```diff
@@ -0,0 +1,20 @@
+# Travis CI (MIT License) configuration file for PubLibData application
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+  # aliased to a recent 5.5.x version
+  - 7.0
+  - 5.6
+
+before_script:
+  - composer install
+
+script:
+  # Search for PHP syntax errors.
+  - find . \( -name '*.php' \) -exec php -lf {} \;
+  # PHPUnit for testing
```

Comment:
Cool! We could also run `phpcs`  using the same find output. https://github.com/squizlabs/PHP_CodeSniffer